autopts/wid/pacs.py

```diff
@@ -29,6 +29,26 @@ def pacs_wid_hdl(wid, description, test_case_name):
     return generic_wid_hdl(wid, description, test_case_name, [__name__])
 
 
+def BIT(bit):
+    return 1 << bit
+
+
+class AUDIO_CONTEXT_TYPE:
+    PROHIBITED = 0
+    UNSPECIFIED = BIT(0)
+    CONVERSATIONAL = BIT(1)
+    MEDIA = BIT(2)
+    GAME = BIT(3)
+    INSTRUCTIONAL = BIT(4)
+    VOICE_ASSISTANTS = BIT(5)
+    LIVE = BIT(6)
+    SOUND_EFFECTS = BIT(7)
+    NOTIFICATIONS = BIT(8)
+    RINGTONE = BIT(9)
+    ALERTS = BIT(10)
+    EMERGENCY_ALARM = BIT(11)
```

Comment:
IMO, those should be placed in  `pybtp/defs.py` just like `PACS_AUDIO_LOCATION_*` definitions